src/pages/newplant.js

```diff
@@ -50,54 +66,59 @@ export default function NewPlant() {
-                        </div>
-                        <div class='field'>
-                            <div class='label'>Temperature</div>
-                            <div class='control is-expanded'>
-                                <input class='input' type='text' id='tempLevel' placeholder='Add Temperature Care'></input>
-                                <span class='icon is-left'></span>
-                                {/* add icon */}
+                            <div class='control'>
+                                <button class='button is-small'>Add New Species</button>
                             </div>
                         </div>
-                        <div class='field'>
-                            <div class='label'>Description</div>
-                            <div class='control is-expanded'>
-                                <textarea class='input' type='text' id='description' placeholder='Description'></textarea>
-                            </div>
+                    </div>
+                    <div className='speciesInfo'>
+                        <div class='context mt-2 has-text-weight-bold'></div>
+                        <ul>
+                            {Object.entries(oneSpec).map(function (el) {
+                                const [key, value] = el;
+                                return (
+                                    <li>
+                                        <span class='context has-text-weight-bold'>{key}</span> : {value}
+                                    </li>
+                                );
+                            })}
+                        </ul>
+                    </div>
+                </form>
+                <form>
+                    <div class='field mt-5'>
+                        <div class='label'>Name</div>
+                        <div class='control is-expanded'>
+                            <input class='input' type='text' id='name' placeholder='Name Your Plant'></input>
                         </div>
-                        <div class='field'>
-                            <div class='control'>
-                                <a class='button'>Submit</a>
-                            </div>
+                    </div>
+                    <div class='field'>
+                        <div class='label'>Last Watered</div>
+                        <div class='control is-expanded'>
+                            <input class='input' type='date' id='date' placeholder='Enter Date'></input>
                         </div>
                     </div>
+                    <div class='field'>
+                        <div class='label'>Add image component here</div>
+                        <figure class='image is-128x128'>
+                            <img src='/sun.png'></img>
+                        </figure>
+                    </div>
+
+                    <div class='control'>
+                        <button class='button is-large'>Add Plant</button>
```

Comment:
you never actually submit the form from what I can tell so it doesn't actually add the plant to the db. When I go back to the home page the plant that I added doesn't appear

Reply:
adding new plant is mostly done, waiting for how we're going to do images